content/docs/lists-and-keys.md

```diff
@@ -70,9 +70,10 @@ ReactDOM.render(
 );
 ```
 
-When you run this code, you'll be given a warning that a key should be provided for list items. A "key" is a special string attribute you need to include when creating lists of elements. We'll discuss why it's important in the next section.
+जब आप यह कोड चलाते हैं, तो आपको एक चेतावनी दी जाती है कि लिस्ट आइटम के लिए एक key प्रदान की जानी चाहिए। "key" एक विशेष स्ट्रिंग विशेषता है जिसे आपको 
+एलिमेंटस की सूची बनाते समय शामिल करने की आवश्यकता होती है। हम चर्चा करेंगे अगले भाग में कि यह महत्वपूर्ण क्यों है।
```

Comment:
`हम चर्चा करेंगे अगले भाग में कि यह महत्वपूर्ण क्यों है।` => `हम अगले भाग में चर्चा करेंगे कि यह महत्वपूर्ण क्यों है।`

how about this?